pulser-core/pulser/json/abstract_repr/schemas/sequence-schema.json

```diff
@@ -445,6 +442,100 @@
       ],
       "type": "object"
     },
+    "OpDisableEOM": {
+      "additionalProperties": false,
+      "properties": {
+        "channel": {
+          "$ref": "#/definitions/ChannelName",
+          "description": "The name of the channel to take out of EOM mode."
+        },
+        "op": {
+          "const": "disable_eom_mode",
+          "type": "string"
+        }
+      },
+      "required": [
+        "op",
+        "channel"
+      ],
+      "type": "object"
+    },
+    "OpEOMPulse": {
+      "additionalProperties": false,
+      "properties": {
+        "channel": {
+          "$ref": "#/definitions/ChannelName",
+          "description": "The name of the channel to add the pulse to."
+        },
+        "duration": {
+          "$ref": "#/definitions/ParametrizedNum",
+          "description": "The duration of the pulse (in ns)."
+        },
+        "op": {
+          "const": "add_eom_pulse",
+          "type": "string"
+        },
+        "phase": {
+          "$ref": "#/definitions/ParametrizedNum",
+          "description": "The pulse phase (in radians)."
+        },
+        "post_phase_shift": {
+          "$ref": "#/definitions/ParametrizedNum",
+          "description": "A phase shift (in radians) immediately after the end of the pulse."
+        },
+        "protocol": {
+          "description": "Stipulates how to deal with eventual conflicts with other channels.",
+          "enum": [
+            "min-delay",
+            "no-delay",
+            "wait-for-all"
+          ],
+          "type": "string"
+        }
+      },
+      "required": [
+        "op",
+        "channel",
+        "duration",
+        "phase",
+        "post_phase_shift",
+        "protocol"
+      ],
+      "type": "object"
+    },
+    "OpEnableEOM": {
```

Comment:
So in order to use the EOM, one has to
1. enable the EOM
2. Create EOMPulse operations
3. Disable the EOM.
Is there a reason we don't automatically enable/disable it on behalf of the user instead of an explicit switch ?

Reply:
Enabling and disabling will automatically add a buffer time necessary to switch in and out of the EOM mode, so we leave it to the user to decide when they're done adding EOM pulses. Plus, all EOM pulses share the parameters specified upon enabling the EOM mode (ie the amplitude and detuning), so it's useful that they are specified at a single point rather than on each pulse.

Reply:
Ok makes sense, thanks